Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:3.10-slim
```

python 3.10 is no longer supported, use python:3.11:


```suggestion
FROM python:3.11-slim
```